scilpy/tractanalysis/scoring.py

```diff
@@ -25,10 +25,11 @@
     - OL : percentage of ground truth voxels containing VS streamline(s).
     - OR/ORn: percentage of voxels containing VS streamline(s) when it
         shouldn't. We compute two versions of the overreach:
-        OR = % of the recovered bundle. Values range between 0 and 100%. Values
-           are not defined when we recovered no streamline for a bundle, but we
-           set the OR to 0 in that case.
-        ORn = % of the ground truth bundle. Values could be higher than 100%.
+        OR_pct_vs = % of the recovered bundle. Values range between 0 and 100%.
```

Comment:
Is overreach ever not a percentage ? If so, maybe the `pct` suffix is redundant ?

Reply:
Yes we do, the count of overreach voxels